src/transformers/models/segformer/configuration_segformer.py

```diff
@@ -148,3 +153,24 @@ def __init__(
         self.decoder_hidden_size = decoder_hidden_size
         self.reshape_last_stage = kwargs.get("reshape_last_stage", True)
         self.semantic_loss_ignore_index = semantic_loss_ignore_index
+
+
+class SegformerOnnxConfig(OnnxConfig):
+
+    torch_onnx_minimum_version = version.parse("1.11")
+
+    @property
+    def inputs(self) -> Mapping[str, Mapping[int, str]]:
+        return OrderedDict(
+            [
+                ("pixel_values", {0: "batch", 1: "num_channels"}),
```

Comment:
The output for `semantic-segmentation` defined in `src/transformers/onnx/config.py` has dynamic axes for height and width, but here the input does not. Have you tested this works with inputs of different sizes?

Reply:
I agree - we should probably have something like the following:
```suggestion
                ("pixel_values", {0: "batch", 1: "num_labels", 2: "height", 3: "width"}),
```

See the "improved" `semantic-segmentation` feature from https://github.com/huggingface/transformers/pull/18587 - this will be merged today

Reply:
Do you mean `num_channels` instead of `num_labels`?

Cause `num_labels` is relevant for the output logits, but not for the inputs.

Reply:
Oh yes, I meant `num_channels` :)

Your implementation LGTM! 

Let's wait for final approval from @sgugger before merging